package.json

```diff
@@ -24,21 +24,21 @@
   "scripts": {
     "test": "grunt test"
   },
+  "dependencies": {
```

Comment:
Assuming this is because consumers need these types and will not need to specify them in the future :)

Reply:
Yes. For a library that's essentially just a `devDependency`, it makes sense to depend on other things that are only `devDependency`s.